docs/content/guide/function.md

```diff
@@ -54,9 +54,10 @@ Spica provides modules to your function in runtime. Modules work like a module i
 
 In order to use these modules in a **function**, they need to be added as **dependency** on **Function Edit page**.
 
-| Module                   | Description                                                                                                  |
-| ------------------------ | ------------------------------------------------------------------------------------------------------------ |
-| `@spica-devkit/database` | This module has a public API for making database operations like **update**, **delete**, **create**, **get** |
+| Module                   | Description                                                                                                                     |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------- |
+| `@spica-devkit/database` | This module has a public API for making database operations like **update**, **delete**, **create**, **get**                    |
+| `@spica-devkit/bucket`   | This module has a public API for making both Bucket and Bucket Data operations like **update**, **delete**, **insert**, **get** |
```

Comment:
Maybe we could add `@spica-devkit/dashboard` as well?

Reply:
Then we have to move the content in dashboard.md to this document. Is it ok for you?

Reply:
I am not sure about that. I think we better leave it as is for now